internal/ci/ci.go

```diff
@@ -81,3 +94,179 @@ func GetProvider() Provider {
 
 	return None
 }
+
+func getCI(provider Provider) CI {
+	var ci CI
+	if reflect.DeepEqual(provider, AppVeyor) {
+		ci = CI{
+			Provider:  provider,
+			OriginURL: os.Getenv("APPVEYOR_URL"),
```

Comment:
Appveyor has no link to the pipeline/job? That would surprise me 🤔

Reply:
I checked the docs, seems no https://www.appveyor.com/docs/environment-variables/

Reply:
Well I get that there's no environment variable with the full link. Neither does github. But we construct one for github. Is it possible to do the same for appveyor?